core/src/components/cat-tooltip/cat-tooltip.tsx

```diff
@@ -0,0 +1,68 @@
+import {Component, h, Host, Prop} from '@stencil/core';
+import {autoUpdate, computePosition, flip, offset, Placement} from "@floating-ui/dom";
+
+@Component({
+  tag: 'cat-tooltip',
+  styleUrl: 'cat-tooltip.scss',
+  shadow: true,
+})
+export class CatTooltip {
+  private tooltip?: HTMLElement;
+  private trigger?: Element;
+
+  /**
+   * The content of the tooltip
+   */
+  @Prop() content!: string;
+
+  /**
+   * Specifies that the tooltip should be disabled. A disabled tooltip is unusable,
+   * and invisible. Corresponds with the native HTML disabled attribute.
+   */
+  @Prop() disabled = false;
+
+  /**
+   * The placement of the tooltip.
+   */
+  @Prop() placement: Placement = 'top';
+
```

Comment:
I forgot two more properties to adjust the delay: `showDelay` and `hideDelay`. Please add them